roles/ssh_hardening/tasks/disable-systemd-socket.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Remove systemd-socket file
+  ansible.builtin.file:
+    path: /etc/systemd/system/ssh.service.d/00-socket.conf
+    state: absent
+  notify: Restart sshd
+
+- name: Enable normal sshd start
+  ansible.builtin.systemd:
+    name: ssh.service
+    state: started
+    enabled: true
+  notify: Restart sshd
+
+- name: Disable systemd-socket activation
+  ansible.builtin.systemd:
+    name: ssh.socket
+    state: stopped
+    enabled: false
+  notify: Restart sshd
```

Comment:
Why did you put the `notify` statements here? I think they are unneccessary.

Also when trying to start ssh before stopping the socket, will this work? (I'm starting to thin my idea was not so great here)

Reply:
Yeah, they're probably unnecessary. I'll remove them.